cli/replicaset/cconfig.go

```diff
@@ -32,21 +41,33 @@ type cconfigTopology struct {
 	InstanceRW bool
 }
 
+// cconfigInstCtx describes an instance context in the cluster config.
+type cconfigInstCtx struct {
+	failover       Failover
+	groupName      string
+	replicasetName string
+	name           string
+}
+
 // CConfigInstance is an instance with the centralized config orchestrator.
 type CConfigInstance struct {
 	evaler connector.Evaler
+	// timeout is a timeout for all operations in seconds.
+	// Keep int, because it can be passed to the target instance.
+	timeout int
```

Comment:
But now it is used only for `Promote`. Do we need to set the timeout for all requests?

Reply:
Moved to the context.